packages/fiori-freestyle/package.json

```
@@ -19,19 +19,21 @@
         "templates"
     ],
     "dependencies": {
-        "ejs": "^3.1.6",
-        "mem-fs": "^2.1.0",
-        "mem-fs-editor": "^9.0.0",
-        "@sap/ux-ui5-application-template": "workspace:*",
+        "@sap/open-ux-tools-common": "workspace:*",
         "@sap/ux-odata-service-template": "workspace:*",
+        "@sap/ux-ui5-application-template": "workspace:*",
         "@sap/ux-ui5-config": "workspace:*",
-        "@sap/open-ux-tools-common": "workspace:*"
+        "ejs": "^3.1.6",
+        "lodash.clonedeep": "4.5.0",
```

```suggestion
        "lodash": "4.17.21",
```
I think "lodash.clonedeep" is an older approach.